### PR TITLE
Move IPFS installation

### PR DESCRIPTION
**Pull request by johnnymatthews**
Contains a new post: _Move your IPFS installation_. 

Fixes protocollabs/docs#442.

### CODE REVIEW

**Comment by johnnymatthews** [comment]
I'm somewhat proud of that typo.

@hsanjuan this addresses [issue 442](https://github.com/ipfs/docs/issues/442 ) that you brought up recently. If you could give this a once over whenever you get a chance I'd appreciate it. No rush though.

File: docs/how-to/move-ipfs-installation.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
+---
+title: Move an IPFS installation
+---
+
+# Move an IPFS installation
+
+This page walks through how to move your IPFS installation from one system to another. This process is cross-platform compatible and is especially useful for readers who are changing operating systems. The process is to grab the `.ipfs` folder from the _home_ directory of the _donor_ system, and copy it over to _home_ directory on the receiving system.
+
+This process is not a backup procedure; do not treat it as such. Many things can go wrong with this process, most of which relate to _peer IDs_. Since we are duplicating an `.ipfs` installation folder, both IPFS clients have the same peer ID. This issue is ok as long as you delete the old _donor_ installation once the copy is complete. Having two IPFS installations with the same peer ID causes substantial problems and could result in loss of data.
```
Comment (hsanjuan) [inline docs/how-to/move-ipfs-installation.md]: Is `IPFS clients` a term we use across the docs? Otherwise, `IPFS peers` is better.

Comment (hsanjuan) [reply docs/how-to/move-ipfs-installation.md]: `Having two IPFS installations with the same peer ID causes substantial problems and could result in loss of data.`. Let's replace that with: you should never start two peers with the same peer IDs on different places, as they will misbehave badly.

File: docs/how-to/move-ipfs-installation.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+    ```bash
+    cp --recursive --verbose .ipfs ipfs-from-linux
+
+    > '.ipfs' -> 'ipfs-from-linux'
+    > '.ipfs/datastore_spec' -> 'ipfs-backup/data'
+    > ...
+    ```
+
+1. You now have a copy of your IPFS repository within the `ipfs-backup`.
```
Comment (hsanjuan) [inline docs/how-to/move-ipfs-installation.md]: The copy is actually in `ipfs-from-linux` ?

Comment (johnnymatthews) [reply docs/how-to/move-ipfs-installation.md]: Good catch. I originally had separate sections for MacOS and Linux, but figured I'd just throw them in together. Now the instructions say to move  `.ipfs` to `ipfs-backup`.

**Pull request merged**